### PR TITLE
Update group.md

### PR DESCRIPTION
**Pull request by abhijeetsinha**
We are adding a new property to group - isAssignableToRole.

### CODE REVIEW

File: api-reference/v1.0/resources/group.md [hunk 1]
```diff
@@ -111,6 +111,7 @@ This resource supports:
...
 |hasMembersWithLicenseErrors|Boolean|Indicates whether there are members in this group that have license errors from its group-based license assignment. <br><br>This property is never returned on a GET operation. You can use it as a $filter argument to get groups that have members with license errors (that is, filter for this property being true). See an [example](../api/group-list.md).|
 |id|String|The unique identifier for the group. <br><br>Returned by default. Inherited from [directoryObject](directoryobject.md). Key. Not nullable. Read-only.|
+|isAssignableToRole|Boolean|Indicates whether this group can be assigned to an Azure AD role or not. <br><br>This property can only be set while creating the group and is immutable. Only Global Admin and Privileged Role Administrator rolescan set this property. For more information, see [Using a group to manage Azure AD role assignments](https://go.microsoft.com/fwlink/?linkid=2103037)<br><br>Returned by default.|
```
Comment (Lauragra) [inline api-reference/v1.0/resources/group.md]: Do we need a changelog entry for the new property?

Comment (Lauragra) [inline api-reference/v1.0/resources/group.md]: I assume that you will need to update the file in the /beta folder as well to list the new property.

Comment (abhijeetsinha) [reply api-reference/v1.0/resources/group.md]: @dkershaw10, @davidmu1  - Can you please provide guidance? Laura is asking if changelog entry is needed for a new property or not.

Comment (abhijeetsinha) [reply api-reference/v1.0/resources/group.md]: @davidmu1 - Can you please help with this?

**Pull request closed without merging**